backend/src/main/java/com/cruru/club/exception/ClubNameLengthException.java

```diff
@@ -0,0 +1,12 @@
+package com.cruru.club.exception;
+
+import com.cruru.advice.BadRequestException;
+
+public class ClubNameLengthException extends BadRequestException {
+
+    public static final String MESSAGE = "동아리 이름이 비어있거나 %d자 초과입니다. 현재 이름 글자수: %d";
+
+    public ClubNameLengthException(int maxLength, int currentLength) {
+        super(String.format(MESSAGE, maxLength, currentLength));
+    }
```

Comment:
한글과 영어, 특수문자에 따라 글자수가 다르게 들어갈텐데 현재 글자 수를 알려줄 필요가 있나 싶어요!
```suggestion
    public static final String MESSAGE = "동아리 이름은 최대 %d까지 입력 가능합니다";

    public ClubNameLengthException(int maxLength) {
        super(String.format(MESSAGE, maxLength));
    }
```

그리고 입력란이 빈 Exception은 따로 처리해야하지 않을까 싶습니다